generic/ras.py

```diff
@@ -20,6 +20,7 @@
 from avocado import main
 from avocado.utils import process
 from avocado.utils.software_manager import SoftwareManager
+from avocado.core import exceptions
```

Comment:
Please don't use internal APIs. They have no guarantee of stability and are intended for internal usage (well, or hacking), not for test writing. In this case, as you're using it for `TestWarn` exception raising, you can use `self.log.warn()` instead.